src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, print_function
+
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils.committers import get_serialized_event_file_committers
+
+
+PREFERRED_GROUP_OWNERS = 3
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+
+
+@instrumented_task(
+    name="sentry.tasks.process_suspect_commits", default_retry_delay=60 * 5, max_retries=5
```

Comment:
This looks like it will have a max duration of 25 minutes. This task doesn't use any external services, what does waiting 5 minutes help with?

Reply:
You're right. I changed this to 5 seconds for a max duration of 25 seconds.